sphinx/texinputs/Makefile.jinja

```diff
@@ -77,7 +77,7 @@ tar: all-$(FMT)
 	rm -r $(ARCHIVEPREFIX)docs-$(FMT)
 
 gz: tar
-	gzip -9 < $(ARCHIVEPREFIX)docs-$(FMT).tar > $(ARCHIVEPREFIX)docs-$(FMT).tar.gz
+	gzip -n -9 < $(ARCHIVEPREFIX)docs-$(FMT).tar > $(ARCHIVEPREFIX)docs-$(FMT).tar.gz
```

Comment:
Is there a long option for ``-n``? Ideally there would also be a comment explaining why we use ``-n`` so we don't forget why it was added in the future.

A

Reply:
Yes. `--no-name`.

Reply:
Though in this case, using the `--no-name` variant might mis-lead people into thinking that it is about the omitting name rather than the timestamp coupled with it. Added an inline comment to explain.